docs/development_suite/api-console/api-design/overview.md

```diff
@@ -119,6 +122,57 @@ In order to switch to another branch or tag, you just need to click on its name
   </div>
 </div>
 
-### How to save your configuration
+## How to save your configuration
+
+Once the respective desired changes have been made, using the `Save Configuration` button in the branch action popover and fill the save configuration modal form. 
+
+<div style={{display: 'flex', justifyContent: 'center'}}>
+  <div style={{display: 'flex', width: '780px'}}> 
+
+![Save Configuration](./img/save-configuration.jpg)
+
+  </div>
+</div>
+
+In this modal, you can specify the title of your commit along with a short description. 
+
+It's also possible to create a new branch along with the commit, which will contain the current configuration.  
+
+:::tip
+Additional information about the configuration may appear on the modal, highlightning potential warnings about your configuration.
+:::
+
+### Create Tag
+
+To keep track of all changes made to the configuration, you may want to create a **tag** using the corresponding button in the branch management area.  
+
+<div style={{display: 'flex', justifyContent: 'center'}}>
+  <div style={{display: 'flex', width: '340px'}}> 
+
+![Create Tag](./img/create-tag.png)
+
+  </div>
+</div>
+
+:::caution
+Since tags are read-only, it is not possible to save changes starting from a tag. You can either:
+* create a new branch using the tag as reference
```

Comment:
I think it is not possible to create a branch from a tag: from branch creation modal, it is possible only to select one another branch in the Create from field. Inside the Design area, when you are inside a tag, on configuration saving it is only possible to save on a new branch.

Reply:
Hi @albertotessarotto thanks for the catch! I've removed the information